BenchmarkSample/ui/src/main/java/com/example/benchmark/ui/MainActivity.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.benchmark.ui
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+
+class MainActivity : AppCompatActivity() {
+    val adapter = SampleAdapter()
+    lateinit var recyclerView: RecyclerView
```

Comment:
Could we replace this with a synthetic property?

Reply:
Do you mean import activity_main.*? I think we don't generally use that part of the android extensions in samples. If nothing else, it's easier to understand for Java developers.

Reply:
I do think we should try to show what we would like people to write, but I agree that synthetic properties are kind of magical.